packages/dataclass-editor/src/master/ValidationRow.tsx

```diff
@@ -26,21 +16,35 @@ export const rowClassName = (messages: Array<MessageData>) => {
 type ValidationRowProps = {
   row: Row<DataClassField>;
   isReorderable: boolean;
+  updateOrder: (moveId: string, targetId: string) => void;
+  onClick?: React.MouseEventHandler<HTMLTableRowElement> | undefined;
+  onDrag?: React.DragEventHandler<HTMLTableRowElement> | undefined;
```

Comment:
Seems like `onClick` will always be defined. If you pass a function to `onDrag` and handle the logic within (see previous comment) the same applies to `onDrag`.
And I don't think it's necessary to add a question mark `?` and `| undefined`